sql/ddl_api.sql

```diff
@@ -93,24 +95,44 @@ BEGIN
         FROM hypertable_ids INNER JOIN LATERAL _timescaledb_internal.dimension_get_time(hypertable_ids.id) time_dim ON (true);
     EXCEPTION
         WHEN NO_DATA_FOUND THEN
+        -- Q:: show chunks implementation now does this, shall I get rid of this transaction alltogether?
```

Comment:
If you are sure that these errors are correctly thrown then yes.

Reply:
done